### ISSUE TITLE
Several Estimators not resulting in Uncertainty Estimations but Values outside [0,1]

### ISSUE THREAD

**Issue by muelphil** [IINemo/lm-polygraph#284]
Hello again!

I was using the Estimators of your framework to compute uncertainties. However some of the Estimators are putting out very large or very small values, that do not represent uncertainties (not within [0,1]). For example the RenyiNeg Estimator gave by very low negative values. Is this intended?

Specifically, for the RenyiNeg Estimator, is the line 
```
probabilities = softmax(logits / self.temperature, axis=-1)
```

intended? As the result of this line is not a list of probabilities but a list of log probabilities still. There is no call to np.exp to convert them to probabilities, but a logarithm is applied again to these values, indicating to me that something is wrong: `np.log(probabilities)`.

Kind regards

**Comment by ArtemVazh** [comment]
Hi, thank you for your questions!

1. This is the expected behavior of the uncertainty quantification methods. UQ methods are mostly designed to provide rankings from low to high uncertainty instances and are rarely well calibrated in the range [0, 1]. If you need a calibrated score, you can find normalizers here: https://github.com/IINemo/lm-polygraph/tree/main/src/lm_polygraph/normalizers (details on the implementation are given in our [TACL paper](https://arxiv.org/pdf/2406.15627 ) in Section 3).

2. Due to the use of the softmax function, the result will be the the same for logits and log. probs.  

```
temperature = 2
logits = np.random.randn(1, 4)
log_probs = np.log(softmax(logits, axis=-1))
(np.log(softmax(logits / temperature, axis=-1)) == np.log(softmax(log_probs / temperature, axis=-1))).all()
```
`True`
